lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -638,8 +638,23 @@ void applyGlobalCssRulesToSheet(
 }
 
 /// The embedder singleton.
-FlutterViewEmbedder get flutterViewEmbedder => ensureFlutterViewEmbedderInitialized();
+///
+/// [ensureFlutterViewEmbedderInitialized] must be called prior to calling this
+/// getter.
+FlutterViewEmbedder get flutterViewEmbedder {
+  final FlutterViewEmbedder? embedder = _flutterViewEmbedder;
+  assert(() {
+    if (embedder == null) {
+      throw StateError(
+        'FlutterViewEmbedder not initialized. Call `ensureFlutterViewEmbedderInitialized()` '
+        'prior to calling the `flutterViewEmbedder` getter.'
+      );
+    }
+    return true;
+  }());
+  return embedder!;
```

Comment:
This might fail at runtime (where the assert hasn't run). How readable is the dart "unexpected null" error message (maybe compared to making the null-check a runtime one, instead of an assertion?)

Reply:
There's nothing a developer can do to cause it because we handle the bootstrap in the Flutter Tool. So I think it's low risk. Removing the lengthy message from release builds is probably worth more.